src/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml

```diff
@@ -69817,6 +69101,7 @@
             <prop name="label" class="sp800-53a" value="SC-42(02)"/>
             <prop name="sort-id" value="sc-42.02"/>
             <link rel="required" href="#sc-42"/>
+            <link rel="required" href="#sc-42.1"/>
```

Comment:
No action needed. This is a change that ensures that the sibling control enhancement `sc-42.1` is selected when `sc-42.2` is selected.